download.html

```diff
@@ -97,8 +97,10 @@ <h2 id="project_tagline">
 		</p>
 		Other packages and versions are available <a href="#versionList">below</a>. For an overview of the current maintenance state of the different OpenUI5 versions, please check the <a href="https://openui5.hana.ondemand.com/versionoverview.html">version overview page</a>.
 
-		<!--<h3>Latest unstable development build:</h3>
-		Currently there is no newer preview release available. It will be added, as 1.50 development progresses.-->
+		<div id="noUnstableVersion" style="display: none">
+			<h3>Latest unstable development build:</h3>
+			Currently there is no newer preview release available. It will be added, as 1.50 development progresses.
```

Comment:
instead of "1.50" there should be {{versionNext}}

Reply:
Done